src/fontcache.h

```diff
@@ -65,6 +65,8 @@ class FontCache {
 	 */
 	virtual int GetFontSize() const { return this->height; }
 
+	virtual void SetFontSize([[maybe_unused]]int pixels);
```

Comment:
```suggestion
	virtual void SetFontSize([[maybe_unused]] int pixels);
```

a few other places too

Reply:
I found 1 other place to fix.